test/testdata/library_gapic.yaml

```diff
@@ -1,29 +1,30 @@
 type: com.google.api.codegen.ConfigProto
 #language_settings: <snip>
 #license_header: <snip>
+config_schema_version: 1.0.0
 collections:
-  - name_pattern: archives/{archive_id}/books/{book_id=**}
-    entity_name: archived_book
+  - name_pattern: archives/{archive}/books/{book}
+    entity_name: archive_book
 collection_oneofs:
   - oneof_name: book_oneof
     collection_names:
     - book
-    - archived_book
+    - archive_book
 collections:
-- name_pattern: projects/{project_id}
+- name_pattern: projects/{project}
   entity_name: project
   language_overrides:
   - language: java
     common_resource_name: com.google.cloud.ProjectName
-- name_pattern: shelves/{shelf_id}
+- name_pattern: shelves/{shelf}
   entity_name: shelf
-- name_pattern: shelves/{shelf_id}/books/{return} # Test using a Java keyword ("return").
+- name_pattern: shelves/{shelf}/books/{book}
   entity_name: book
   language_overrides:
     - language: java
       entity_name: shelf_book
-- name_pattern: archives/{archive_id}/books/{book_id=**}
-  entity_name: archived_book
+- name_pattern: archives/{archive}/books/{book}
+  entity_name: archive_book
```

Comment:
Could we have a language_override for java for `archive_book` to `archived_book` so there is no diff, or a smaller one, in the baseline file?

Reply:
Just looked into this, but the problem is then there is a difference between the v1 and v2 config cases, and the goal was to try to make them use the same set of baseline filed.

Reply:
Actually, I did make this work by adding it to the v2 config as well.

Reply:
ok, so long as it is possible